workflow/metrics/scripts/run.py

```diff
@@ -102,6 +102,9 @@ def replace_last(source_string, replace_what, replace_with, cluster_key='leiden'
 adata.obs = adata.obs[cluster_columns+[batch_key, label_key]].copy()
 adata.obs.rename(columns=lambda x: replace_last(x, '_1', ''), inplace=True)
 
+if 'feature_name' in adata.var.columns:
+    adata.var_names = adata.var['feature_name']
+
```

Comment:
Would be best to call this right after reading the anndata object instead of here